README.md

```diff
@@ -67,7 +67,12 @@ to embed one of several language runtimes on top of its key-value store. Clients
 
 ## Getting Started on Azure Confidential Computing
 
-Under `getting_started/`:
```

Comment:
I think this should say "if you are checking out the repo", because some/most users will download release tarballs.